menu-screen.lua

```diff
@@ -91,32 +100,34 @@ function MenuScreen:update(dt)
 end
 
 function MenuScreen:draw()
+	self:drawShinyTitle()
+
 	self.font = love.graphics.newFont("font/Retro Computer_DEMO.ttf", 20)
-	love.graphics.setFont(self.font);
+	love.graphics.setFont(self.font)
 	gPrint(self.subTitle, love.graphics.getWidth()/2 - self.font:getWidth(self.subTitle) / 2, love.graphics.getHeight()/4)
 	self.font = love.graphics.newFont("font/Retro Computer_DEMO.ttf", 14)
-	love.graphics.setFont(self.font);
+	love.graphics.setFont(self.font)
```

Comment:
this is weird. we should store which fonts we use on init, not generate them on runtime:

> This function can be slow if it is called repeatedly, such as from love.update or love.draw. If you need to use a specific resource often, create it once and store it somewhere it can be reused!